packages/discovery-provider/src/tasks/index_trending.py

```diff
@@ -178,7 +178,9 @@ def index_trending(self, db: SessionManager, redis: Redis, timestamp):
     top_trending_tracks = get_top_trending_to_notify(db)
 
     index_trending_notifications(db, timestamp, top_trending_tracks)
-    index_tastemaker_notifications(db, top_trending_tracks)
+    index_tastemaker_notifications(
```

Comment:
nit would be good to just rename this something like `index_tastemaker` or even throw it into the trending indexer lol